app/components/publications/overview/publications-table-row.hbs

```diff
@@ -40,9 +40,15 @@ request the whole row and not only the arrow at the end to be clickable }}
   {{#if @tableColumnDisplayOptions.shortTitle}}
     <td data-test-publication-table-row-short-title class="auk-table__col--6">
       <Auk::AbbreviatedText
+        class="auk-o-flex"
         @text={{@publicationFlow.shortTitle}}
         @maxLength={{150}}
-      />
+        as |text|
+      >
+        <div class="auk-o-flex__item">
```

Comment:
In geval bovenstaande comment resolved wordt en we er voor kiezen om `auk-o-flex` intern aan de component te houden, lijkt het me het eenvoudigst van ook het `auk-o-flex__item` intern aan de component te houden.
Zoals nu, met de helft van de CSS-component intern aan de ember-component en de andere helft van de CSS-component extern aan de ember-component, vind ik het nogal moeilijk te begrijpen

Reply:
Ik denk ook dat dit niet ideaal is. De reden dat ik dit niet gedaan heb is dat het uitlijnen van de ellipsen me eerder een uitzondering lijkt.
Ik ben ook niet overtuigd dat die `Auk::AbbreviatedText` de beste oplossing is. Er is minder flexibiliteit in verschillen in weergave dan nodig.
Dit blijkt ook uit de verschillende marges onder /publicaties en onder /dossiers

> #### /dossiers
> ![dossier-ellips](https://user-images.githubusercontent.com/80692792/153380303-0b2bb45c-1f0b-4dc6-ae15-00b2fd44cc31.png)

> #### /publicaties
> ![publicatie-ellips](https://user-images.githubusercontent.com/80692792/153380307-ddffe8f9-8475-47f3-9881-4b0a8beb4368.png)
> 

(Die titel-bar zou overigens in zijn geheel consistenter mogen. Ik weet ook niet of het correct is dat die border onder /publicaties niet over gans de breedte loopt. @brenner-company?)

Reply:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1191/commits/a5a7871349d0ffdbe5b1d5fb6988b4bd1ed3a3e2#diff-8b3fd437558cdc93d15f5139ac86df222b98ced64a8e8f2114ee2157a2ca5c6aL43 verwijderd